### ISSUE TITLE
(5P) Help Button: Focus on Contact button for Screen Readers

### ISSUE THREAD

**Issue by donalirl** [Automattic/wp-calypso#41506]
There is a usability issue blocking customers from contacting support when using a screen reader.

#### Steps to reproduce

1. Starting at WordPress.com while logged in, use a screen reader to navigate to the Help button
2. Tap the help button
3. Use a left to right swiping gesture to navigate through the options and click the Contact button
4. Attempt to send a message to the support team

#### What I expected

The voiceover cursor would focus on the contact text box so I can start writing my message to the support team. I should then be able to submit that message to access chat or send an email (depending on if chat is available or not.)

#### What happened instead

Voiceover cursor focuses on the entire browser window again. A visually-impaired user cannot navigate to the Contact option to send a message.

#### Request

1. Upon tapping Help → Contact Us, can the cursor focus on the contact form? [**essential**]
2. Upon tapping the Help button, can the cursor focus on Contact Us first, instead of the support articles like it does now? [**nice to have**]

We also need to test the flow to ensure a user with a screen reader can access support through the Help button.

#### Browser / OS version
Chrome (latest), Windows 10
Chrome (latest) on android
Safari (latest) on iPhone

#### Screenshot / Video

Screen recording: https://d.pr/v/0Fps4T

#### Context / Source

#p4vt7e-6n-p2

#journey #empathy #livesharing #accessibility #painpoint #user-report

**Comment by davemart-in** [comment]
@obenland and @jeryj another one related to the help container.
